src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -1581,6 +1591,13 @@ def test_notification_push_kafka():
     notification_push('kafka', conn)
 
 
+@attr('kafka_test')
```

Comment:
this should be a manual test, if we run that in teuthology it won't work as we do not have the othjer brokers running.
if you want to make sure we have regression for the feature, you should add multiple brokers to: `qa/tasks/kafka.py`

Reply:
let'me try!